### DISCUSSION TITLE
Stop notifications from already taken-down repo

### DISCUSSION THREAD

**Discussion by samuelawachie** [community/community#130983] [Repositories]
### Select Topic Area

General

### Body

I setup my github to watch a repository (https://github.com/notifications?query=repo%3Aarichornlover%2FuYouEnhanced) amongst others of course. But that particular repo was taken down a few days ago. Since then I have removed it from my notification list. I have unwatched it, but I'm still getting the blue notification bubble on my profile, and it lists its notification as 1. There is no way for me to visit said notification to "clear" it, and the whole repo is not coming back up for the foreseeable future. 
I hate seeing that blue notification badge and having no way to clear it. What do I do to fix this please because it's quite triggering for me..

**Comment by queenofcorgis** [comment]
Hi @samuelawachie,

Thanks for sharing this. Do you have any screenshots you can share?

**Reply by samuelawachie** [reply]
> 
> Thanks for sharing this. Do you have any screenshots you can share?
>
Here's the screenshot you requested.
![Screenshot from 2024-07-08 18-26-35](https://github.com/community/community/assets/65021017/08ba7826-d163-4ca7-a3a6-1be5454daea5)

**Comment by oldman20** [comment]
I have exactly same this case, include take down repo

**Comment by oldman20** [comment]
any solution now, sorry?

**Reply by samuelawachie** [reply]
> any solution now, sorry?
>
Nothing, as of yet. GitHub have shown from their response and actions they don't care about this. That's all I have to say about that at the moment, I'm afraid.

**Reply by oldman20** [reply]
It is so disappointing and too troublesome
